Core/Graphics/Include/Android/Babylon/GraphicsPlatform.h

```diff
@@ -0,0 +1,13 @@
+ #pragma once
+
+#include <android/native_window.h>
+
+using WindowType = ANativeWindow*;
+
+struct GraphicsConfiguration
+{
+    WindowType windowPtr;
+    void* windowTypePtr;
```

Comment:
Can't this be removed now since there is a platform specific function for translating from `GraphicsConfiguration` to bgfx platform data?

Reply:
Yeah we can define GraphicsConfiguration differently in the different platform headers now, I'll change that.

Reply:
Changed.